aspnetcore/includes/vs-mac-eol.md

```diff
@@ -0,0 +1,14 @@
+---
+author: tdykstra
+ms.author: tdykstra
+ms.date: 09/15/2023
+ms.topic: include
+---
```

Comment:
Should we include this meta-data with includes? We haven't been doing that.

Reply:
Theoretically it's required, even more than I put in: https://review.learn.microsoft.com/en-us/help/platform/metadata-required-include-files?branch=main